shared/actions/fs/index.js

```diff
@@ -482,7 +482,7 @@ const extractMimeFromContentType = (contentType, disposition: string): Types.Mim
   return Constants.makeMime({displayPreview, mimeType})
 }
 
-const getMimeTypePromise = (localHTTPServerInfo: Types._LocalHTTPServer, path: Types.Path) =>
+const getMimeTypePromise = (localHTTPServerInfo: Types.LocalHTTPServer, path: Types.Path) =>
```

Comment:
these inner types shouldn't ever be used (only by the make functions in constants) and not leaked